app.js

```diff
@@ -1,9 +1,11 @@
 const express = require('express');
 const app = express();
-// const fs = require('fs/promises');
 const getTopics = require('./controllers/topics.controllers');
+const endpoints = require('./endpoints.json');
 
-app.use(express.json());
+app.get('/api', (req, res) => {
+  res.status(200).send({ endpoints: endpoints });
```

Comment:
would be good to make use of a controller here (move this function into your controller.js and require in). but altogether good!